server/Dockerfile

```
@@ -39,4 +39,4 @@ VOLUME /usr/src/app/upload
 
 EXPOSE 3001
 
-ENTRYPOINT ["/bin/sh"]
+ENTRYPOINT ["tini", "--"]
```

I think this can be changed to avoid modifying the docker compose file. The default entry point is "/bin/sh -c", so it should remain that way.


```suggestion
ENTRYPOINT ["tini", "--", "/bin/sh", "-c"]
```

---

The problem is how the CMD is passed:

```sh
docker inspect immich_server
```

```
[
    {
        "Id": "4978ed236c4578e552e72b276d7e6739e25e3a3c6d786f4bd699697206923d9e",
        "Created": "2023-07-21T13:04:46.1112611Z",
        "Path": "tini",
        "Args": [
            "--",
            "/bin/sh",
            "-c",
            "start.sh",
            "immich"
        ],
```

The result:
```sh
tini -- /bin/sh -c start.sh immich
```

```
immich: line 0: start.sh: not found
```

There are two problems:
1. the command passed to `-c` needs to be a single string like `'start.sh immich'`
2. `start.sh` is not on the path so this would still yield `/bin/sh: start.sh: not found`

---

I'll test without `-c`

---

Works :slightly_smiling_face: